### DISCUSSION TITLE
Erro de integração com o Asaas

### DISCUSSION THREAD

**Discussion by CaSoDev** [RamonSilva20/mapos#2564] [Q&A]
Estou com esse problema aqui na versão 4.42.0
Já coloquei a API no application/config/payment_gateways.php
Já autorizei o IP dentro do Asaas mas mesmo assim permanece com o erro
![Captura de tela 2024-10-16 134804](https://github.com/user-attachments/assets/d2ddc46b-e705-484a-a44b-c38300c49736)

**Comment by willph** [comment]
Baixou a versão mais recente do app?

https://github.com/RamonSilva20/mapos/archive/refs/heads/master.zip

Tem a correção desse erro.

Em qua., 16 de out. de 2024, 13:49, S0MBR413 ***@***.***>
escreveu:

**Reply by CaSoDev** [reply]
Já tentei utilizar e permaneceu o mesmo erro
